src/isomorphic/children/__tests__/onlyChild-test.js

```diff
@@ -94,4 +94,12 @@ describe('onlyChild', () => {
     }).not.toThrow();
   });
 
+  it('should return the only child', () => {
+    var instance =
```

Comment:
Normally every time you want to start working on a PR you should pull the master branch from our repo and create a new local branch from it. You are likely seeing this because you forgot to switch back to master before creating the second feature branch.